pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -1194,6 +1205,24 @@ pj_status_t pjsua_vid_channel_update(pjsua_call_media *call_med,
         if (status != PJ_SUCCESS)
             goto on_error;
 
+        /* Temporarily pause the encoding to avoid early RTP packet lost
+         * because media transports are being setup/attached to streams
+         * and CPU may be spiking after SDP nego.
+         */
+        status = pjmedia_vid_stream_pause(call_med->strm.v.stream,
+                                          PJMEDIA_DIR_ENCODING);
+        if (status != PJ_SUCCESS)
+            goto on_error;
+
+        /* Schedule start stream encoding */
+        if (acc->cfg.vid_out_auto_transmit) {
+            pjmedia_stream_common *c_strm = (pjmedia_stream_common*)
+                                            call_med->strm.v.stream;
+            pj_grp_lock_add_ref(c_strm->grp_lock);
+            pjsua_schedule_timer2(&timer_on_start_vid_stream_encoding,
+                                  call_med->strm.v.stream, 500);
```

Comment:
as usual, configurable is better. at least local configurable, with 0 meaning disabled/old behavior.